app/controllers/wrangling_guidelines_controller.rb

```diff
@@ -9,7 +8,7 @@ def index
 
   # GET /wrangling_guidelines/1
   def show
-    @wrangling_guideline = WranglingGuideline.find(params[:id])
+    @wrangling_guideline = WranglingGuideline.find(wrangling_guideline_id)
```

Comment:
I'm just starting to do some reading, but everything I've read so far continues to use the old `find(params[:id])` approach here while switching to strong parameters for `new` and `update_attributes`. Do you have a link that explains why it's a good idea to the new approach for `find` as well?